courses/models_test.py

```diff
@@ -343,6 +343,49 @@ def test_is_future(self):
         )
         assert course_run.is_future is False
 
+    def test_is_future_enrollment_open(self):
+        """Test for is_future_enrollment_open property"""
+        # with no start date
+        course_run = self.create_run()
+        assert course_run.is_future_enrollment_open is False
+
+        # with start in the future, no enrollment_start
+        course_run = self.create_run(
+            start=self.now + timedelta(weeks=2)
+        )
+        assert course_run.is_future_enrollment_open is False
+
+        # enrollment_start in the past, no enrollment_end
```

Comment:
Nevermind, we should be doing separate functions but this way it's consistent with the rest of the file